src/System.Net.HttpListener/tests/HttpListenerPrefixCollectionTests.cs

```diff
@@ -12,6 +12,7 @@ namespace System.Net.Tests
     public class HttpListenerPrefixCollectionTests
     {
         [Fact]
+        [ActiveIssue(17462, TargetFrameworkMonikers.Uap)]
```

Comment:
> [ActiveIssue]  now can be used in a test class to disable it complete.

Thanks.  :)  I was aware of that. But sometimes adding them to each test method makes it easier for the next developer to start turning the tests back on, one test method at a time.